Toolkit/ArcGISToolkitTests/ArcGISToolkitTests.swift

```diff
@@ -63,3 +63,19 @@ class ArcGISToolkitTests: XCTestCase {
         XCTAssertNotNil(bookmarksVC)
     }
 }
+
+/// Helper function to load an `AGSLoadable` object, waiting until it's loaded to return.
+/// - Parameter object: The loadable object.
+public func XCTLoad(_ object: AGSLoadable) {
+    // Wait for the object to load.
+    let loadExp = XCTestExpectation(description: "expectation for `object.load`")
+    object.load { (error) in
+        XCTAssertNil(error)
+        loadExp.fulfill()
+    }
+    let waitResult = XCTWaiter().wait(for: [loadExp], timeout: 5.0)
+    guard waitResult == .completed else {
```

Comment:
This seems like an odd use of `guard` since it doesn't halt forward execution; the same thing happens (the function returns) both immediately after the statement and inside the `else` clause. One idea is to just use `XCTAssertEqual`:

```swift
XCTAssertEqual(waitResult, .completed, file: file, line: line)
```